lte/gateway/python/integ_tests/s1aptests/test_secondary_pdn_reject_multiple_sessions_not_allowed_per_apn.py

```diff
@@ -0,0 +1,78 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import unittest
+import time
+
+import s1ap_types
+import s1ap_wrapper
+
+
+class TestSecondaryPdnRejectMultipleSessionsNotAllowedPerApn(
+    unittest.TestCase
+):
+    def setUp(self):
+        self._s1ap_wrapper = s1ap_wrapper.TestWrapper()
+
+    def tearDown(self):
+        self._s1ap_wrapper.cleanup()
+
+    def test_secondary_pdn_reject_multiple_sessions_not_allowed_per_apn(self):
+        """ Attach a single UE + send standalone PDN connectivity
+        request with the same apn as the default apn + attach reject
+        + detach"""
+        num_ue = 1
+
+        self._s1ap_wrapper.configUEDevice(num_ue)
+        req = self._s1ap_wrapper.ue_req
+        ue_id = req.ue_id
+
+        print(
+            "************************* Running End to End attach for UE id ",
+            ue_id,
+        )
+        # Attach
+        self._s1ap_wrapper.s1_util.attach(
+            ue_id,
+            s1ap_types.tfwCmd.UE_END_TO_END_ATTACH_REQUEST,
+            s1ap_types.tfwCmd.UE_ATTACH_ACCEPT_IND,
+            s1ap_types.ueAttachAccept_t,
+        )
+
+        # Wait on EMM Information from MME
+        self._s1ap_wrapper._s1_util.receive_emm_info()
+
+        # Send PDN Connectivity Request
+        apn = "magma.ipv4"
+        self._s1ap_wrapper.sendPdnConnectivityReq(ue_id, apn)
+        # Receive PDN Connectivity reject
+        response = self._s1ap_wrapper.s1_util.get_response()
+        self.assertEqual(
+            response.msg_type, s1ap_types.tfwCmd.UE_PDN_CONN_RSP_IND.value
```

Comment:
how is this assert checking rejection?

Reply:
In s1ap tester uePdnConRsp_t is a common message for handling Activate default EPS bearer context request/PDN connectivity reject